README.markdown

```diff
@@ -215,14 +215,14 @@ For UIKit view controllers, consider grouping lifecycle, custom accessors, and I
 
 ### Unused Code
 
-Unused (dead) code, including Xcode template code and placeholder comments should be removed. An exception is when your tutorial or book instructs the user to use the commented code.
+Unused (dead) code, including Xcode template code and placeholder comments should be removed.
```

Comment:
Add comma after comments:

Unused (dead) code, including Xcode template code and placeholder comments, should be removed.